content/en/docs/Writing policies/match-exclude.md

```diff
@@ -238,6 +238,8 @@ spec:
     - name: match-criticals-except-given-rbac
       match:
         resources:
+	        kinds:
```

Comment:
The red block is still there 😅 are you able to select that portion and remove it?

Reply:
@RinkiyaKeDad , can you please check. I think I have removed the red portion.